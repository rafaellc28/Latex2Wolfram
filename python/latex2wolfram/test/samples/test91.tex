% Test erro in differential variable declaration
y^{(1.2)}(t)
