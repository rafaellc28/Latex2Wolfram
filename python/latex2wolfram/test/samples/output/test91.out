Syntax error at line 2, position 5: '1.2'.
Context: y^{(1.2)}(t).
